lunes_cms/cms/forms.py

```diff
@@ -26,6 +26,34 @@ def label_from_instance(self, obj):
             return obj.title
 
 
+class DocumentForm(forms.ModelForm):
+    """
+    This is the form for the Document model
+    """
+
+    class Meta:
+        """
+        Defining the meta description for the `DocumentForm`
+        """
+
+        model = Document
+        fields = ["word_type", "plural", "plural_article"]
+
+        def clean(self):
+            word_type = self.clean_fields.get("word_type")
+
+            if word_type == "Substantiv":
+                self.fields_required(["plural"])
+            else:
+                self.clean_fields["plural"] = ""
+
+    def fields_required(self, fields):
+        for field in fields:
+            if not self.cleaned_data.get(field, ""):
```

Comment:
```suggestion
            if not self.cleaned_data.get(field):
```
`dict.get()` already returns `None` when the fields does not exist, so no need for an explicit default value.